lib/sqlalchemy/orm/mapped_collection.py

```diff
@@ -38,28 +49,35 @@ class _PlainColumnGetter:
 
     __slots__ = ("cols", "composite")
 
-    def __init__(self, cols):
+    def __init__(self, cols: List[KeyedColumnElement[Optional[_KT]]]) -> None:
         self.cols = cols
         self.composite = len(cols) > 1
 
-    def __reduce__(self):
+    def __reduce__(
+        self,
+    ) -> Tuple[
+        Type[_SerializableColumnGetterV2],
+        Tuple[List[Tuple[str, Optional[_KT]]]],
+    ]:
         return _SerializableColumnGetterV2._reduce_from_cols(self.cols)
 
-    def _cols(self, mapper):
+    def _cols(
+        self, mapper: Mapper[_KT]
+    ) -> List[KeyedColumnElement[Optional[_KT]]]:
         return self.cols
 
-    def __call__(self, value):
+    def __call__(self, value: _KT) -> Union[_KT, Tuple[_KT, ...]]:
         state = base.instance_state(value)
         m = base._state_mapper(state)
 
-        key = [
+        key = [  # type: ignore
```

Comment:
Again, for some reason, mypy is silent here for me after removing all ignores :) python 3.10.9 and mypy 0.991 - just in case